source/draft_api.rst

```diff
@@ -131,6 +131,21 @@ GET Endpoints
       "id": 1
     }
 
+In addition, the endpoint at `/times` also supports several querystring parameters: user,
+project, activity, and date range. These are accessed via `/times?user=:username`,
+`/times?project=:projectslug`, `/times?activity=:activityslug`, `/times?start=:date`, and
+/times?end=:date (note that dates are in ISO-8601 format). When multiple different
+parameters are used, they narrow down the result set (for example,
+`/times?user=example-user&activity=dev` will return all time entries which were entered by
+example-user AND which were spent doing development). When the same parameter is repeated,
+they expand the result set (for example, `/times?activity=gwm&activity=pgd` will return all
+time entries which were either for gwm OR pgd). Date ranges are inclusive on both ends.
+
+If a query parameter is provided with a bad value (e.g. invalid slug, or date not in ISO
+8601 format), a Bad Query Value error is returned. If a query parameter with a nonexistent
+name is provided, it is ignored. If multiple `start` or `end` parameters are provided,
```

Comment:
This is kind of ambiguous: "If a query parameter with a nonexistent name is provided, it is ignored."
Maybe: "Any query parameter other than those specified in this document will be ignored."